homeassistant/components/lutron/event.py

```diff
@@ -0,0 +1,102 @@
+"""Support for Lutron events."""
+from enum import StrEnum
+
+from pylutron import Button, Keypad, Lutron, LutronEvent
+
+from homeassistant.components.event import EventEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_ID
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util import slugify
+
+from . import ATTR_ACTION, ATTR_FULL_ID, ATTR_UUID, DOMAIN, LutronData
+from .entity import LutronKeypad
+
+
+class LutronEventType(StrEnum):
+    """Lutron event types."""
+
+    SINGLE_PRESS = "single_press"
+    PRESS = "press"
+    RELEASE = "release"
```

Comment:
I think we could use a better name for the event types.

Reply:
They look pretty sane to me?